server/db/seeds/01_seeds.sql

```diff
@@ -34,13 +39,36 @@ INSERT INTO events (name, image, description, size_limit, owner_id, category, la
 
 ('Early Morning Hiking', 'https://images.unsplash.com/photo-1551632811-561732d1e306?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8aGlraW5nfGVufDB8fDB8fA%3D%3D&w=1000&q=80', 'come for a nice hike with me but we have to get up at 3am to see sunrise on top of mountain', 10, 4, 5, 49.37341791072464, -123.09637524954998, '2022-09-29 18:00:00', '2022-09-29 20:00:00'),
 
-('Lets talk Shakespeare', 'https://calhum.org/wp-content/uploads/2017/07/AP_Blog_Enewsjpg.jpg', 'Come discuss classic literature with me and some friends', 5, 5, 6, 49.28053130736995, -123.11693171231532, '2022-09-20 13:00:00', '2022-09-20 14:00:00'),
+('Chess Club', 'https://c4.iggcdn.com/indiegogo-media-prod-cld/image/upload/c_fill,w_695,g_auto,q_auto,dpr_2.6,f_auto,h_460/gxknjgscx8ci7ayvxxve', 'Come play Chess with us! Try new strategies, discuss tactics and meet people. Free admission', 8, 5, 2, 49.282126, -123.124361, '2022-10-13 18:00:00', '2022-10-13 21:00:00'),
 
```

Comment:
did you update the category numbers on the events as well to match the new categories?

Reply:
Sure did! Category 2 is for bgames

Reply:
Oh poo - I just checked again and there are a few mis-categorized options - I'll fix em in a new commit